src/main/java/com/linkedin/kmf/KafkaMonitor.java

```diff
@@ -101,17 +101,13 @@ public synchronized void start() {
       entry.getValue().start();
     }
 
-    _executor.scheduleAtFixedRate(
-      new Runnable() {
-        @Override
-        public void run() {
-          try {
-            checkHealth();
-          } catch (Exception e) {
-            LOG.error("Failed to check health of tests and services", e);
-          }
-        }
-      }, 5, 5, TimeUnit.SECONDS
+    _executor.scheduleAtFixedRate(() -> {
+      try {
+        checkHealth();
+      } catch (Exception e) {
+        LOG.error("Failed to check health of tests and services", e);
+      }
+    }, 5, 5, TimeUnit.SECONDS
```

Comment:
Nit: can you make these two variables for these two "5" ? I think it would increase the readability by a little
```java
long initialDelaySecond = 5;
long periodSecond = 5;

_executor.scheduleAtFixedRate(() -> {
       try {
         checkHealth();
       } catch (Exception e) {
         LOG.error("Failed to check health of tests and services", e);
       }
     }, initialDelaySecond, periodSecond, TimeUnit.SECONDS
```

Reply:
Amended and resolved.